ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json

```diff
@@ -0,0 +1,94 @@
+{
+  "resourceType": "ServiceRequest",
+  "id": "ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example",
+  "extension": [
+    {
+      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-Coverage",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-FundingCategory",
+            "code": "nhs",
+            "display": "NHS"
+          }
+        ]
+      }
+    }
+  ],
+  "status": "active",
+  "intent": "reflex-order",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-GenomeSequencingCategory",
+          "code": "rare-disease-non-wgs",
+          "display": "Rare Disease - Non-WGS"
+        }
+      ]
+    }
+  ],
+  "priority": "routine",
+  "code": {
+    "coding": [
+      {
+        "system": "https://fhir.nhs.uk/CodeSystem/England-GenomicTestDirectory",
+        "code": "R59",
+        "display": "Early onset or syndromic epilepsy"
+      }
+    ]
+  },
+  "subject": {
+    "reference": "Patient/Patient-DemeizaSeo-Example",
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9449306559"
+    }
+  },
+  "authoredOn": "2023-11-13",
+  "requester": {
+    "reference": "PractitionerRole/PractitionerRole-GeneSmithENT-Example"
+  },
+  "reasonCode": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.nhs.uk/CodeSystem/England-GenomicTestDirectory",
+          "code": "R387",
+          "display": "Re-analysis"
```

Comment:
Here, might need to look at either using a SNOMED code (such as 719641000000101 | Data analysis (qualifier value) |) or proposing an addition to SNOMED (as this is how diagnostic/carrier have been coded in previous examples)

Reply:
Could also add a text element here "Significant new treatment and or clinical management implication for the patient, treatment for management of epilepsy" (as this is the reason specified for requesting reanalysis)